Stopping dev_appserver on Windows always fails with com.intellij.execution.ExecutionException

This should have always been the case for every run of dev_appserver.

com.intellij.execution.ExecutionException: Cannot run program "true": CreateProcess error=2, The system cannot find the file specified
	at com.google.cloud.tools.intellij.appengine.server.run.CloudSdkStartupPolicy$2$1.createProcessHandler(CloudSdkStartupPolicy.java:136)
	at com.intellij.javaee.run.execution.PatchedLocalState$ScriptBasedLocalJavaeeServerProcessHandler.shutdown(PatchedLocalState.java:154)
	at com.intellij.javaee.run.execution.LocalJavaeeServerProcessHandler.destroyProcessImpl(LocalJavaeeServerProcessHandler.java:162)
	at com.intellij.javaee.run.execution.LocalJavaeeServerProcessHandler.detachProcessImpl(LocalJavaeeServerProcessHandler.java:167)
	at com.intellij.execution.process.ProcessHandler$2.run(ProcessHandler.java:125)
	at com.intellij.execution.process.ProcessHandler$TasksRunner.execute(ProcessHandler.java:259)
	at com.intellij.execution.process.ProcessHandler.detachProcess(ProcessHandler.java:120)
	at com.intellij.javaee.run.execution.LocalJavaeeServerProcessHandler.destroyProcess(LocalJavaeeServerProcessHandler.java:78)
	at com.intellij.javaee.run.execution.J2EEProcessHandlerWrapper$4.onProcess(J2EEProcessHandlerWrapper.java:194)
	at com.intellij.javaee.run.execution.J2EEProcessHandlerWrapper$ProcessCall.lambda$new$0(J2EEProcessHandlerWrapper.java:360)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.intellij.util.concurrency.BoundedTaskExecutor$2.run(BoundedTaskExecutor.java:210)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.IOException: Cannot run program "true": CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
	at com.google.cloud.tools.intellij.appengine.server.run.CloudSdkStartupPolicy$2$1.createProcessHandler(CloudSdkStartupPolicy.java:133)
	... 15 more
Caused by: java.io.IOException: CreateProcess error=2, The system cannot find the file specified
	at java.lang.ProcessImpl.create(Native Method)
	at java.lang.ProcessImpl.<init>(ProcessImpl.java:386)
	at java.lang.ProcessImpl.start(ProcessImpl.java:137)
	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
	... 16 more
Disconnected from server

--- a/google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java
+++ b/google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java
@@ -127,14 +127,7 @@ public class CloudSdkStartupPolicy implements ExecutableObjectStartupPolicy {
           public OSProcessHandler createProcessHandler(
               String workingDirectory, Map<String, String> envVariables) throws ExecutionException {
             startupProcessHandler.destroyProcess();
-
-            ProcessBuilder dummyProcess = new ProcessBuilder("true");
-            try {
-              return new OSProcessHandler(dummyProcess.start(),
-                  GctBundle.getString("appengine.run.shutdownscript"));
-            } catch (IOException ioe) {
-              throw new ExecutionException(ioe);
-            }
+            return null;
           }
         };
       }

--- a/google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java
+++ b/google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java
@@ -34,7 +34,6 @@ import com.intellij.javaee.run.localRun.ScriptsHelper;
 
 import org.jetbrains.annotations.Nullable;
 
-import java.io.IOException;
 import java.util.Arrays;
 import java.util.Map;
 

